libreyolo/backends/base.py

```
@@ -944,6 +1227,97 @@ def _parse_dfine(self, all_outputs, orig_w, orig_h, conf, max_det: int = 300):
         mask = scores > conf
         return boxes[mask], scores[mask], class_ids[mask].astype(np.int64)
 
+    def _parse_ec_segment(self, all_outputs, orig_w, orig_h, conf, max_det=300):
```

Ruff (0.15.17)

[warning] Missing return type annotation for private function `_parse_ec_segment`

(ANN202)